share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/guest/login.get.html.ftl

```diff
@@ -30,7 +30,23 @@
          <div class="error">${msg("message.loginautherror")}</div>
          <#else>
          <script type="text/javascript">//<![CDATA[
-            document.cookie = "_alfTest=_alfTest; Path=/;";
+            <#assign cookieHeadersConfig = config.scoped["COOKIES"] />
+            <#if cookieHeadersConfig?? && (cookieHeadersConfig.secure.getValue() == "true" || cookieHeadersConfig.secure.getValue() == "false")>
```

Comment:
Here, to improve the OR condition between true and false, a better solution could be to add an utility method parseboolean to be sure of returning true when is true and false when is something else than true.